orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -325,18 +347,31 @@ def backward(ctx, *grad_outputs):
             self._torch_alloc = self._torch_cuda_allocator.cuda_caching_allocator_raw_alloc_address()
             self._torch_free = self._torch_cuda_allocator.cuda_caching_allocator_raw_delete_address()
 
+    @property
+    def is_training(self):
+        return self._is_training and torch.is_grad_enabled()
+
+    @is_training.setter
+    def is_training(self, training_mode):
+        self._is_training = training_mode
+
     def _initialize_module_gradient_graph_builder(self):
         # TODO: PyTorch exporter bug: changes the initializer order in ONNX model
-        initializer_names = [p[0]
-                             for p in self._flattened_output_module.named_parameters()]
+        initializer_names = [name
+                             for name, _ in self._flattened_output_module.named_parameters()]
+        initializer_names_to_train = []
+        if torch.is_grad_enabled():
```

Comment:
should it be `self.is_training`?

Reply:
yeah, I think this should be is_training. Updating.